src/pages/Lending/Pool/components/Repay/RepayInput.tsx

```diff
@@ -317,30 +317,35 @@ export const RepayInput = ({
 
   const swapIcon = useMemo(() => <ArrowDownIcon />, [])
 
-  const { data: lendingSupportedAssets, isLoading: isLendingSupportedAssetsLoading } =
+  const { data: supportedRepaymentAssets, isLoading: isSupportedRepaymentAssetsLoading } =
```

Comment:
Same as above, added repaymentAssetIds, renamed for sanity, and colocated data and derived data to keep things clean